packages/edit-site/src/components/page-pages/index.js

```diff
@@ -347,85 +377,33 @@ export default function PagePages() {
 
 	// TODO: we need to handle properly `data={ data || EMPTY_ARRAY }` for when `isLoading`.
 	return (
-		<>
-			<Page
-				className={
-					view.type === LAYOUT_LIST
-						? 'edit-site-page-pages-list-view'
-						: null
-				}
-				title={ __( 'Pages' ) }
-				actions={
-					<>
-						<Button variant="primary" onClick={ openModal }>
-							{ __( 'Add new page' ) }
-						</Button>
-						{ showAddPageModal && (
-							<AddNewPageModal
-								onSave={ handleNewPage }
-								onClose={ closeModal }
-							/>
-						) }
-					</>
-				}
-			>
-				<DataViews
-					paginationInfo={ paginationInfo }
-					fields={ fields }
-					actions={ actions }
-					data={ pages || EMPTY_ARRAY }
-					isLoading={ isLoadingPages || isLoadingAuthors }
-					view={ view }
-					onChangeView={ onChangeView }
-					onSelectionChange={ onSelectionChange }
-					onDetailsChange={ onDetailsChange }
-				/>
-			</Page>
-			{ view.type === LAYOUT_LIST && (
```

Comment:
All what's after that has been removed, it now corresponds to the default "preview" area rendered in the Layout.